### ISSUE TITLE
command line help info not match 

### ISSUE THREAD

**Issue by Micah2015** [machinezone/IXWebSocket#97]
I found ws cmd help info SUBCOMMAND after options, but in fact the cmd is `./ws SUBCOMMAND [OPTIONS]`
``` 
./ws --help                             
ws is a websocket tool
Usage: ./ws [OPTIONS] SUBCOMMAND
```

**Comment by bsergean** [comment]
Hey @Micah2015 I'm not sure I understand your question too well, could you rephrase it?

ws has subcommands (ws connect) like git (git init).
